script/deploy/DeployChainlinkRelayers.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.18;
+
+import { console } from "forge-std-next/console.sol";
+import { Script } from "script/utils/mento/Script.sol";
+import { Chain } from "script/utils/mento/Chain.sol";
+import { Contracts } from "script/utils/mento/Contracts.sol";
+
+import { ChainlinkRelayerFactory } from "mento-core-develop/oracles/ChainlinkRelayerFactory.sol";
+import { ChainlinkRelayerFactoryProxy } from "mento-core-develop/oracles/ChainlinkRelayerFactoryProxy.sol";
+import { ChainlinkRelayerFactoryProxyAdmin } from "mento-core-develop/oracles/ChainlinkRelayerFactoryProxyAdmin.sol";
+import { IChainlinkRelayer } from "mento-core-develop/interfaces/IChainlinkRelayer.sol";
+
+contract DeployChainlinkRelayers is Script {
+  using Contracts for Contracts.Cache;
+
+  struct Relayer {
+    string rateFeed;
+    IChainlinkRelayer.Config config;
+  }
+
+  Relayer[] relayers = [
+    Relayer({
+      rateFeed: "chainlink:CELO/USD:v1",
+      config: singleAggConfig(contracts.dependency("Chainlink.CELOUSD"), false)
+    }),
+    Relayer({
+      rateFeed: "chainlink:USDT/USD:v1",
+      config: singleAggConfig(contracts.dependency("Chainlink.USDTUSD"), false)
+    }),
+    Relayer({
+      rateFeed: "chainlink:CELO/PHP",
+      config: IChainlinkRelayer.Config({
+        maxTimestampSpread: 1000,
+        chainlinkAggregator0: contracts.dependency("Chainlink.CELOUSD"),
+        chainlinkAggregator1: contracts.dependency("Chainlink.PHPUSD"),
+        chainlinkAggregator2: address(0),
+        chainlinkAggregator3: address(0),
+        invertAggregator0: false,
+        invertAggregator1: true,
+        invertAggregator2: false,
+        invertAggregator3: false
+      })
+    }),
+    Relayer({ rateFeed: "PHP/USD", config: singleAggConfig(contracts.dependency("Chainlink.PHPUSD"), false) })
```

Comment:
grouping-wise would feel more logical to move PHP/USD above CELO/PHP, then it'd read:
1. Deploy CELO/USD
2. Deploy PHP/USD
3. Compose both feeds into CELO/PHP